i6_models/assemblies/conformer/conformer.py

```diff
@@ -0,0 +1,115 @@
+from __future__ import annotations
+
+__all__ = ["ConformerBlockV1Config", "ConformerEncoderV1Config", "ConformerBlockV1", "ConformerEncoderV1"]
+
+import torch
+from torch import nn
+from dataclasses import dataclass
+
+from i6_models.config import ModelConfiguration, SubassemblyWithOptions
+from i6_models.parts.conformer import (
+    ConformerConvolutionV1,
+    ConformerConvolutionV1Config,
+    ConformerMHSAV1,
+    ConformerMHSAV1Config,
+    ConformerPositionwiseFeedForwardV1,
+    ConformerPositionwiseFeedForwardV1Config,
+)
+
+
+@dataclass
+class ConformerBlockV1Config(ModelConfiguration):
+    """
+    Attributes:
+        ff_cfg: Configuration for ConformerPositionwiseFeedForwardV1
+        mhsa_cfg: Configuration for ConformerMHSAV1
+        conv_cfg: Configuration for ConformerConvolutionV1
+    """
+
+    # nested configurations
+    ff_cfg: ConformerPositionwiseFeedForwardV1Config
+    mhsa_cfg: ConformerMHSAV1Config
+    conv_cfg: ConformerConvolutionV1Config
+
+
+class ConformerBlockV1(nn.Module):
+    """
+    Conformer block module
+    """
+
+    def __init__(self, cfg: ConformerBlockV1Config):
+        """
+        :param cfg: conformer block configuration with subunits for the different conformer parts
+        """
+        super().__init__()
+        self.ff_1 = ConformerPositionwiseFeedForwardV1(cfg=cfg.ff_cfg)
```

Comment:
I think it is more consistent (standard) to use `ff1` instead of `ff_1`.
```suggestion
        self.ff1 = ConformerPositionwiseFeedForwardV1(cfg=cfg.ff_cfg)
```
Similarly for `ff2` below.